playground/frontend/playground_components/lib/src/widgets/overlay/opener.dart

```diff
@@ -19,17 +19,19 @@
 import 'package:flutter/material.dart';
 
 import '../../controllers/public_notifier.dart';
-import 'dismissible.dart';
+import 'widget.dart';
 
 void openOverlay({
   required BuildContext context,
   required PublicNotifier closeNotifier,
   required Positioned positioned,
+  bool isDismissible = true,
```

Comment:
Why not `false`?

Reply:
The current users of overlay (LoginButton and Avatar) expect it to be dismissible. Also, it is consistent with showDialog's `bool barrierDismissible = true`.

Reply:
Rename it to `barrierDismissible` then?
I would prefer this to be required though because there is actually no good reason to prefer one over another.

Reply:
I renamed the function to `showOverlay` and the parameter to `barrierDismissible` for consistency, still assuming that `true` is the preferable setting.